src/test/java/it/pagopa/transactions/client/PaymentGatewayClientTest.java

```diff
@@ -324,7 +324,7 @@ void shouldThrowAlreadyProcessedOn401ForCreditCardWithXpay() throws JsonProcessi
         StepVerifier.create(client.requestXPayAuthorization(authorizationData))
                 .expectErrorMatches(
                         error -> error instanceof AlreadyProcessedException &&
-                                ((AlreadyProcessedException) error).getRptId()
+                                ((AlreadyProcessedException) error).getTransactionId()
                                         .equals(transaction.getNoticeCodes().get(0).rptId())
```

Comment:
Here should equality be tested against transactions id and not 1st notices RPT ID?

Reply:
(you're right, see below, fixed in c84d279)